src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -242,6 +246,10 @@ internal static bool DoSslHandshake(SafeSslHandle context, byte[] recvBuf, int r
 
         internal static int Encrypt(SafeSslHandle context, ReadOnlyMemory<byte> input, ref byte[] output, out Ssl.SslErrorCode errorCode)
         {
+#if DEBUG
```

Comment:
I missed this when I separated a larger PR into smaller clean ups for SSL. I will add something to capture the whole error queue but perhaps not on this PR.